mono/metadata/debug-mono-ppdb.c

```diff
@@ -172,8 +172,7 @@ mono_ppdb_load_file (MonoImage *image, const guint8 *raw_contents, int size)
 		return NULL;
 	}
 
-// Temporarily disabled to unblock Roslyn
-#if HOST_WIN32 || HOST_WASM
+#if HOST_WIN32 || HAVE_SYS_ZLIB
```

Comment:
Are embedded pdbs always compressed?

Reply:
I think yes

Reply:
Don't you also need the `HOST_WASM` here? I thought wasm *doesn't* have a system zlib?